assests/JavaScript/logic.js

```diff
@@ -0,0 +1,225 @@
+let correct = 0
+let incorrect = 0
+let unanswered = 0
+let index = 0
+let randomMovieArray = []
+let moviePool = movieDataBase.length
+let status = true
+let m = -1
```

Comment:
Try your best to not name variables with just single letters unless it is painfully obvious. Reading your code, I don't know what `m` does or represents.